ckanext/advancedauth/templates/user/new_user_form.html

```diff
@@ -109,6 +109,12 @@
   </div>
   <div class="form-actions">
     {% block form_actions %}
+    {% set sitekey = "3x00000000000000000000FF" if h.is_running_in_dev() else "0x4AAAAAAAkj_0rhuMVRWMMj"%}
```

Comment:
What are these values?

Reply:
The `3x...FF` is a development key https://developers.cloudflare.com/turnstile/troubleshooting/testing/ . The `0x4...Mj` is my sitekey. The retrieved token is then validated on the backend with the secret key.